scripts/apis_scan.py

```diff
@@ -298,7 +299,14 @@ def generate_report(scan_timestamp):
 
 if __name__ == "__main__":
 
-    workDir = resultDir + sys.argv[1] + "/"
+    parser = argparse.ArgumentParser(description='Connect to ZAP and launch a scan based on config.yaml')
+    parser.add_argument('--debug', action='count', default=0, help='Show debugging output')
```

Comment:
Do you think a `--log-level` with list of choices would give some more freedom here?

Reply:
Possibly, I originally just wanted a quick way to make INFO level comments visible by default, and leave room for future improvements.